---a/net/minecraft/world/WorldSavedData.java
+++b/net/minecraft/world/WorldSavedData.java
@@ -2,14 +2,14 @@
 
 import net.minecraft.nbt.NBTTagCompound;
 
-public abstract class WorldSavedData
-{
+public abstract class WorldSavedData {
+
     public final String mapName;
     private boolean dirty;
     private static final String __OBFID = "CL_00000580";
 
-    public WorldSavedData(String name)
-    {
+    public WorldSavedData(String name) {
         this.mapName = name;
     }
 
@@ -17,19 +17,15 @@
 
     public abstract void writeToNBT(NBTTagCompound nbt);
 
-    public void markDirty()
-    {
+    public void markDirty() {
         this.setDirty(true);
     }
 
-    public void setDirty(boolean isDirty)
-    {
+    public void setDirty(boolean isDirty) {
         this.dirty = isDirty;
     }
 
-    public boolean isDirty()
-    {
+    public boolean isDirty() {
         return this.dirty;
     }
-}
+}
